lib/stat_tracker.rb

```diff
@@ -180,6 +181,16 @@ def most_tackles(season_id)
   def fewest_tackles(season_id)
     @teams.find{|team| team.team_id == count_tackles(season_id).min_by {|team_id, tackles| tackles}.first}.teamname
   end
+
+  def best_season(team_id)
+    # team_season_average = {}
```

Comment:
we can erase this now?

Reply:
I beleive so. I'll clean it up on the next push!